### ISSUE TITLE
VerifyError - ShadowGenerator

### ISSUE THREAD

**Issue by ZieIony** [ZieIony/Carbon#134]
This issue was originally posted by @gorodechnyj in #127:

And in my case this leads to crash on API 15 emulator. The crash happens when I open DrawerLayout. I also use ActionBarDrawerToggle with it:

```
actionBarDrawerToggle = new android.support.v7.app.ctionBarDrawerToggle(...
drawerLayout.setDrawerListener(actionBarDrawerToggle);
actionBarDrawerToggle.syncState();
```

09-02 19:09:42.803    9592-9592/com.masani E/AndroidRuntime﹕ FATAL EXCEPTION: main
    java.lang.VerifyError: carbon/shadow/ShadowGenerator
            at carbon.widget.LinearLayout.getShadow(LinearLayout.java:489)

**Comment by ZieIony** [comment]
This issue was solved in #40

**Comment by ZieIony** [comment]
No answer for more than a week. I'm closing the issue.
